disnake/ui/select/base.py

```diff
@@ -187,9 +184,6 @@ def _create_decorator(
     if (origin := get_origin(cls)) is not None:
         cls = origin
```

Comment:
With the subtype guard removed this conversion is no longer necessary, it was needed for `isinstance`/`issubclass` to work as they reject `GenericAlias`es

On another note, it shouldn't hurt to add a `if callable(cls)` guard

Reply:
hh021450@gmail.com